src/flake8_tidy_imports/settings.rs

```diff
@@ -50,6 +78,17 @@ impl Default for Settings {
     fn default() -> Self {
         Self {
             ban_relative_imports: Strictness::Parents,
+            banned_api: FxHashMap::default(),
+        }
+    }
+}
+
+impl Hash for Settings {
+    fn hash<H: Hasher>(&self, state: &mut H) {
+        self.ban_relative_imports.hash(state);
+        for (k, v) in &self.banned_api {
```

Comment:
I think we may need to sort this by key to ensure a stable hash. Or use a `BTreeMap`.